tests/unit/copying_manager_tests/common.py

```diff
@@ -938,6 +938,8 @@ def workers_log_processors_count(self):
         six.ensure_str("get_checkpoints_path"),
         six.ensure_str("_init_scalyr_client"),
         six.ensure_str("change_agent_log"),
+        six.ensure_str("get_checkpoints"),
+        six.ensure_str("write_checkpoints"),
```

Comment:
I'm still curious as to why `_init_scalyr_client` has to be proxied, and now also curious about `get_checkpoints` and `write_checkpoints`.  Do those all have to be invoked across the multiprocess boundary?

Reply:
Oh, right, it shouldn't be there. But we have to expose other functions in order to be able to test multi-process workers. Those methods are only members of the "testable" subclass of the worker to make testing easier.